tools/realsense-viewer/openvino-face-detection.cpp

```diff
@@ -213,6 +213,7 @@ class openvino_face_detection : public post_processing_worker_filter
         _frameset_log_info = false;
         // A color video frame is the minimum we need for detection
         auto cf = fs.get_color_frame();
```

Comment:
This line may cause undefined behavior when the input is rs2::frame

Reply:
changed to:
auto cf = f;
if (fs) cf = fs.get_color_frame();